cityofaustin/janis
# 4185 make janis preview

# Description
service page and official documents page previews should be working







this joplin is needed for api updates https://github.com/cityofaustin/joplin/pull/666

# How can this be tested?



 
     
* pull this branch down
* get it running (I used start-joplin-local)
* go here http://localhost:3000/en/preview/services/UGFnZVJldmlzaW9uTm9kZTox?CMS_API=https://joplin-pr-fixtures-and-preview.herokuapp.com/api/graphql and see it working

# Checklist:
